### ISSUE TITLE
Automation failure : test_put_status_code_11_Host

### ISSUE THREAD

**Issue by tkolhar** [SatelliteQE/robottelo#2319]
```
TestNames:
tests.foreman.api.test_multiple_paths.EntityIdTestCase.test_put_status_code_11_Host 

Error Message
422 Client Error: Unprocessable Entity

Stacktrace
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel71-api/tests/foreman/api/test_multiple_paths.py", line 396, in test_put_status_code
    entity_id = entity_cls().create_json()['id']
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 689, in create_json
    response = self.create_raw(create_missing)
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 670, in create_raw
    self.create_missing()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entities.py", line 1598, in create_missing
    self.ptable = PartitionTable(self._server_config).create(True)
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 719, in create
    return self.read(attrs=self.create_json(create_missing))
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 690, in create_json
    response.raise_for_status()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/requests/models.py", line 851, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
'422 Client Error: Unprocessable Entity\n--------------------
```

**Comment by tkolhar** [comment]
when executed locally using nosetests:
$ nosetests -c robottelo.properties -m test_put_status_code_11_Host /home/tkolhar/git/robottelo/tests/foreman/api/test_multiple_paths.py
/usr/lib/python2.7/site-packages/nose/plugins/manager.py:395: RuntimeWarning: Unable to load plugin xunitmp = nose_xunitmp:XunitMP: cannot import name force_unicode
  RuntimeWarning)
Failure: AttributeError ('module' object has no attribute 'DockerComputeResource') ... ERROR

ERROR: Failure: AttributeError ('module' object has no attribute 'DockerComputeResource')
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/loader.py", line 413, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/tkolhar/git/robottelo/tests/foreman/api/test_multiple_paths.py", line 23, in <module>
    entities.ContentView, entities.DockerComputeResource,
AttributeError: 'module' object has no attribute 'DockerComputeResource'

XML: foreman-results.xml
Ran 1 test in 0.001s
FAILED (errors=1)

**Comment by abalakh** [comment]
Test failed due to known issue - [BZ1229384](https://bugzilla.redhat.com/show_bug.cgi?id=1229384).

**Comment by abalakh** [comment]
While issue is not fixed, test may occasionally fail - when ```PartitionTable``` with name of 1 character is generated.
Temporary fix on robottelo side would require change on each PartitionTable creation call in all the tests, and temporary fix on nailgun side is not applicable as nailgun should not depend from satellite. So please ignore such failures while [BZ1229384](https://bugzilla.redhat.com/show_bug.cgi?id=1229384) is not yet fixed.

**Comment by tkolhar** [comment]
this issue occurred again dated 07/08/2015

TestNames:
tests.foreman.api.test_multiple_paths.EntityIdTestCase.test_put_status_code_11_Host 

Error Message
422 Client Error: Unprocessable Entity

Stacktrace

  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel71-api/tests/foreman/api/test_multiple_paths.py", line 406, in test_put_status_code
    entity.create_missing()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entities.py", line 1679, in create_missing
    self.ptable = PartitionTable(self._server_config).create(True)
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 837, in create
    return self.read(attrs=self.create_json(create_missing))
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/nailgun/entity_mixins.py", line 808, in create_json
    response.raise_for_status()
  File "/home/jenkins/shiningpanda/jobs/f593b203/virtualenvs/d41d8cd9/lib/python2.7/site-packages/requests/models.py", line 851, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
'422 Client Error: Unprocessable Entity\n--------------------

**Comment by Ichimonji10** [comment]
The most recent failure was triggered by Robottelo attempting to create a partition table with name one character long. @oshtaier has filed a bug for this issue in [BZ 1229384](https://bugzilla.redhat.com/show_bug.cgi?id=1229384). The problematic character is [Unicode Han Character 'U+223E1'](http://www.fileformat.info/info/unicode/char/223e1/index.htm). It's a valid UTF-8 character that's 32 bits long.

**Comment by Ichimonji10** [comment]
This should be fixed now that #2583 is merged.

**Comment by elyezer** [comment]
Test is passing now, closing this.
